extraConfigSriov.py

```diff
@@ -38,6 +40,33 @@ def _sno_make_deploy(
     client.oc("create namespace openshift-sriov-network-operator")
     client.oc("label ns --overwrite openshift-sriov-network-operator " "pod-security.kubernetes.io/enforce=privileged " "pod-security.kubernetes.io/enforce-version=v1.24 " "security.openshift.io/scc.podSecurityLabelSync=false")
 
+    ocp_version = None
+    v, number_of_subs_made = re.subn("^(4[.][0-9]+).*$", "\\1", version)
+    if number_of_subs_made == 1:
+        if v in ("4.14", "4.15", "4.16", "4.17"):
+            # Only these tags exist currently for the default_base_images
+            ocp_version = v
+    if ocp_version is None:
+        logger.error_and_exit(f"unsupported version \"{version}\"")
```

Comment:
Can we extract a function and call it like this:
`ocp_version = extract_verion_or_panic(version)`

This simplify the type of `ocp_version` and no need to keep thinking about the fact that it might be `None` since the type is explicitly saying it can' be `None`.